site/en/docs/privacy-sandbox/attribution-reporting/web-to-app-attribution/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Web-to-app and app-to-web measurement'
+subhead: >
+  Learn how your Chrome web app can pass attributions to your Android app.  
+description: >
+  Learn how your Chrome web app can pass attributions to your Android app.    
+date: 2023-08-09
+authors:
+  - maudn
+---
+
+The Privacy Sandbox natively supports cross- web and app attribution, and the Attribution Reporting API allows measurement across mobile browsers and Android apps.
+
+## What is web-to-app attribution?
+
+If a user clicks an ad in a Chrome mobile browser, and then goes on to later make a purchase in an Android app, the Attribution Reporting API can directly attribute that conversion performed in the Android app to the ads shown in the Chrome mobile browser. That's web-to-app attribution.
+
+Similarly, if a user clicks an ad in an Android app, and then goes on to later make a purchase in a Chrome mobile browser, the Attribution Reporting API can directly attribute that conversion. That's app-to-web attribution.
+
+Note that the API records web-to-app attributions when they occur on the same device.
```

Comment:
This opens more questions for me. Does this work only if it's on the same device? What if I open an ad on an app, switch to the browser and then open the browser on a different device–with a larger screen–to make the purchase? Does that count as a conversion? How does the device switching affect the other use cases?

Reply:
(Also the ideal answer is probably "the attributions are only counted when they occur on the same device", right?)

Reply:
https://developer.android.com/design-for-safety/privacy-sandbox/attribution#app-web-based-trigger-paths

Reply:
is this not helping? "Note that the API records web-to-app attributions when they occur on the same device." i.e. line 21?

Reply:
Also I'm not sure now if this doc is even needed. Maybe until the move though